File: src/artesgc/scripts/java/ternary.java
package artesgc.scripts.java;

import java.util.Scanner;

public class ternary {
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);

        System.out.print("Digite um numero:\n> ");
        int n1 = input.nextInt();

        System.out.print("Digite outro numero:\n> ");
        int n2 = input.nextInt();

        int maior = n1>n2?n1:n2,  // Math.max(1, 2)
            menor = n1<n2?n1:n2;  // Math.min(1, 2)
        System.out.println("\nO maior valor e -> " + maior + "\nO menor valor e -> " + menor);
    }
}

File: src/artesgc/scripts/kotlin/ternary.kt
package artesgc.scripts.kotlin

import java.util.Scanner

fun main() {
    val input = Scanner(System.`in`)

    print("Digite um numero:\n> ")
    val n1 = input.nextInt()

    print("Digite outro numero:\n> ")
    val n2 = input.nextInt()

    val maior = if (n1 > n2) n1 else n2 // Math.max(1, 2)
    val menor = if (n1 < n2) n1 else n2 // Math.min(1, 2)
    println("\nO maior valor e -> $maior\nO menor valor e -> $menor")
}

